client/app/visualizations/edit-visualization-dialog.js

```diff
@@ -32,6 +38,24 @@ const EditVisualizationDialog = {
       this.visualization = this.newVisualization();
     }
 
+    this.has3plusColumnsFunction = () => {
+      let has3plusColumns = false;
+      if ((JSON.stringify(this.visualization.options.columnMapping).match(/,/g) || []).length > 1) {
+        has3plusColumns = true;
+      }
+      return has3plusColumns;
+    };
+
+    this.disableSubmit = () => {
+      if (this.has3plusColumnsFunction() &&
+        JSON.stringify(this.visualization.options.columnMapping).includes('unused') &&
```

Comment:
I have a query where one of the columns named "unused"... so this is not so reliable.

But this can be tested with `values(this.visualization.options.columnMapping).includes('unused')`.

(`values` is imported from `lodash`)

Reply:
Thanks!